src/prefect/server/events/actions.py

```diff
@@ -1083,6 +1087,40 @@ async def new_state(self, triggered_action: "TriggeredAction") -> StateCreate:
         )
 
 
+class ResumeFlowRun(FlowRunAction):
+    """Resumes a paused or suspended flow run associated with the trigger"""
+
+    type: Literal["resume-flow-run"] = "resume-flow-run"
+
+    async def act(self, triggered_action: "TriggeredAction") -> None:
+        flow_run_id = await self.flow_run(triggered_action)
+
+        self._resulting_related_resources.append(
+            RelatedResource.model_validate(
+                {
+                    "prefect.resource.id": f"prefect.flow-run.{flow_run_id}",
+                    "prefect.resource.role": "target",
+                }
+            )
+        )
+
+        logger.info(
```

Comment:
tiny preference for this to be DEBUG level
```suggestion
        logger.debug(
```

Reply:
Will do!